filebeat/scripts/docs_collector.py

```diff
@@ -1,6 +1,7 @@
 import os
 import argparse
 import yaml
+import six
```

Comment:
why is six also needed here? as this does not seem to be related to the license files